core/configuration/dt_config_file_reader.go

```diff
@@ -58,7 +58,7 @@ type jsonConfigFileReader struct {
 // ReadConfigFromFile looks for a config file "dtconfig.json" in the current directory and attempts to parse it.
 // Returns an error if the file can't be read or the parsing fails.
 func (j *jsonConfigFileReader) readConfigFromFile() (fileConfig, error) {
-	return j.readConfigFromFileByPath("./dtconfig.json")
+	return j.readConfigFromFileByPath("./serverless_function_source_code/dtconfig.json")
```

Comment:
It is unexpected that not obviously cloud-specific core functions use a hard-coded cloud-specific path. Can this maybe be used as a fallback? Or maybe we instead need instructions how to configure package.json to copy the dtconfig.json from source to build output?

Reply:
Since the go exporter is only meant to be used in GCF, this is a quick fix to make it work in GCF only (at the moment it's only documented for GCF, but I believe in future this will be used in other clouds as well). Note that it will be improved in the next iteration so it handle the dtconfig.json more gracefully. I think there are few options here: 1. documenting how to set the path to the file 2. automatically set the path based on the cloud 3. something else

Reply:
As mentioned in #38 , this PR is redundant and should not be merged. I'll decline it.